Returning Instead Of Breaking - Stop Processing Additional View Directories

Fenikkusu opened the pull request:
Hello!

* Type: bug fix
* Link to issue: https://github.com/phalcon/cphalcon/issues/12977 | https://github.com/phalcon/cphalcon/issues/11633

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [x] I wrote some tests for this PR
- [x] I have updated the relevant CHANGELOG
- [ ] I have created a PR for the [documentation](https://github.com/phalcon/docs) about this change

Small description of change:

If you specify multiple view directories using, it creates an abnormality where the same file will be included from every view directory it exists. Rather than continuing the view directory loop, this breaks out of the function completely.

Thanks

niden commented:
cc https://github.com/maurits150 https://github.com/memmaker - thanks to the original reporters of this.

Fenikkusu commented:
I'll get the testing added in tomorrow.

maurits150 commented:
I guess that notExists can be removed now that you always return when a view is found.

Fenikkusu commented:
I've added the testing in for this. Additionally, per @maurits150 , I have also removed the notExists boolean as it isn't needed anymore. I also went ahead and updated the change log.

niden commented:
Excellent work. Thank you @Fenikkusu and @maurits150 for the implementation and input
